kubernetes/AKS/README.md

```diff
@@ -0,0 +1,262 @@
+
+#### 1.1 Set Azure account information
+
+```az login``` 
+
+```az account set -s your-subscription-ID```
+
+#### 1.2 Create a resource group
+
+An Azure resource group is a logical group in which Azure resources are deployed and managed. When you create a resource group, you are asked to specify a location. This location is where resource group metadata is stored, it is also where your resources run in Azure if you don't specify another region during resource creation. Create a resource group using the [az group create](https://docs.microsoft.com/en-us/cli/azure/group#az-group-create) command.
+
+The following example creates a resource group named *myResourceGroup* in the *eastus* location.
+
+```az group create --name myResourceGroup --location eastus```
+
+#### 1.3 Create AKS cluster
+
+Use the [az aks create](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create) command to create an AKS cluster. The following example creates a cluster named *myAKSCluster* with one node. This will take several minutes to complete.
+
+```az aks create  --resource-group myResourceGroup  --name myAKSCluster --node-vm-size Standard_NC6   --node-count 1```
+
+#### 1.4 Connect to the cluster
+
+To manage a Kubernetes cluster, you use [kubectl](https://kubernetes.io/docs/user-guide/kubectl/), the Kubernetes command-line client. If you use Azure Cloud Shell, `kubectl` is already installed. To install `kubectl` locally, use the [az aks install-cli](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli) command:
+
+```az aks install-cli```
+
+To configure `kubectl` to connect to your Kubernetes cluster, use the [az aks get-credentials](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-get-credentials) command. This command downloads credentials and configures the Kubernetes CLI to use them.
+
+```az aks get-credentials --resource-group myResourceGroup --name myAKSCluster```
+
+#### 1.5 Install helm
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+### 2 Deploy TorchServe on AKS
+
+#### 2.1 Download the github repository and enter the kubernetes directory
+
+```git clone https://github.com/pytorch/serve.git```
+
+```cd serve/kubernetes/AKS```
+
+#### 2.2 Install NVIDIA device plugin
+
+Before the GPUs in the nodes can be used, you must deploy a DaemonSet for the NVIDIA device plugin. This DaemonSet runs a pod on each node to provide the required drivers for the GPUs.
+
+```kubectl apply -f templates/nvidia-device-plugin-ds.yaml```
+
+NAME                  READY  STATUS  RESTARTS  AGE
+
+nvidia-device-plugin-daemonset-7lvxd  1/1   Running  0     42s
```

Comment:
nit: Since the EKS document puts log results in boxes, let's do the same to keep the docs consistent.
(Note: I'm unable to actually suggest the change because of how markdown works)
```
NAME                  READY  STATUS  RESTARTS  AGE
nvidia-device-plugin-daemonset-7lvxd  1/1   Running  0     42s
```